### PR TITLE
ArcherTune

### PR DESCRIPTION
**Pull request by Tenscore**
Per request:
Dropped Archer pen down a notch in Axe, Oak, Leng, and UK.
Leaving BL, and BB for their makers as they're still a WiP.

### CODE REVIEW

File: Segments/Oakvael.mapproj [hunk 1]
```diff
@@ -111552,7 +111551,6 @@
         BaseDodge = 14,
         HideDetection = 21,
         Experience = 1755,
-        BasePenetration = ShieldPenetration.VeryLight,
```
Comment (Chilly021) [inline Segments/Oakvael.mapproj:111555]: For Very light removal, you might want to switch that to "none".

File: Segments/Oakvael.mapproj [hunk 2]
```diff
@@ -111398,7 +111398,6 @@
         BaseDodge = 14,
         HideDetection = 21,
         Experience = 1507,
-        BasePenetration = ShieldPenetration.VeryLight,
```
Comment (Chilly021) [inline Segments/Oakvael.mapproj:111401]: For Very light removal, you might want to switch that to none. Otherwise it inherits from weapon.

**Review by Chilly021 — COMMENTED** [review]
Few comments around ShieldPenetration.Verylight. By removing this it will just take into account whatever the weapon has at the time. You can use None. 

BasePenetration = ShieldPenetration.None

[WorldForge]
	public enum ShieldPenetration
	{
		None = 0,

		VeryLight	= 1,
		Light		= 2,
		Medium		= 3,
		Heavy		= 4,
		VeryHeavy	= 5,
		Full		= 6,
	}

File: Segments/Oakvael.mapproj (hunk 2 above)
Comment (Tenscore) [reply Segments/Oakvael.mapproj:111401]: Just noticed Crossbow.cs would set it back to Light.

Nice catch, thx!

File: Segments/Oakvael.mapproj (hunk 1 above)
Comment (Tenscore) [reply Segments/Oakvael.mapproj:111555]: Got it.

**Comment by Tenscore** [comment]
Seems it may be best to attack this at Crossbow.cs, and Longbow.cs

**Review by Leolus — APPROVED** [review]

**Review by zrikzlok — APPROVED** [review]

**Pull request merged**